tools/analyse-storage/src/main.rs

```diff
@@ -0,0 +1,268 @@
+use clap::Parser;
+use near_store::db::RocksDB;
+use near_store::{col_name, DBCol};
+use rayon::prelude::*;
+use rocksdb::{ColumnFamilyDescriptor, Options, DB};
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+use std::{panic, println};
+use strum::IntoEnumIterator;
+
+#[derive(Parser)]
+struct Cli {
+    db_path: String,
```

Comment:
I suggest to make initialization more consistent with our current logic to open DB.

For example, we construct DB path by
* using default value for home dir in `NeardOpts.home`:
```
    #[clap(long, value_parser, default_value_os = crate::DEFAULT_HOME.as_os_str())]
    home: PathBuf,
```
* some logic in `DBOpener::new` choosing hot/cold storage.

So lazy users could just pass Temperature instead of full path. We can assume Temperature::Hot by default.